src/components/Stepper/Stepper.view.tsx

```diff
@@ -0,0 +1,56 @@
+import { ReactNode, useState } from 'react';
+import { styles } from './Stepper.styles.css';
+import StepperProgress from './StepperProgress';
+import useI18n from '../../hooks/useI18n';
+import Button from '~/components/Button';
+
+export type Props = {
+  steps: {
+    title: string;
+    content: ReactNode;
+    tip?: string;
+  }[];
+  onConfirm: () => void;
+  continueLabel?: string;
+  backLabel?: string;
+  confirmLabel?: string;
+};
+
+const Stepper = ({ steps, onConfirm, continueLabel, backLabel, confirmLabel }: Props) => {
+  const { t } = useI18n();
+  const [stepIndex, setStepIndex] = useState(0);
+  const currentStep = steps[stepIndex];
+
+  return (
+    <div className={styles.root}>
+      <StepperProgress
+        step={stepIndex + 1}
+        title={currentStep.title}
+        totalSteps={steps.length}
+        tip={currentStep.tip}
+      />
+      <div className={styles.content}>{currentStep.content}</div>
+      {stepIndex === steps.length - 1 && (
+        <Button onClick={onConfirm} fullWidth>
```

Comment:
Maybe buttons could be separated components like `ConfirmButton`, `BackButton` and `ContinueButton`, to increase readability?

Reply:
I think it wouldn't be beneficial. We still would have to pass `onClick` action and `label` to those separated buttons unless we create them in `Stepper.view.tsx` body which would not be elegant I think.
Also there is no additional logic/design for the buttons - they are just basic `Button` with simple props passed.